composer.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "getkirby/cms",
-	"description": "The Kirby 3 core",
+	"description": "The Kirby 4 core",
```

Comment:
I'd vote to leave out the version number entirely. Then we can't forget it. 😅

Reply:
I think it's a solution that will forever 💯